### PR TITLE
examples: added chat, iframe and pdf components to hello app

### PR DESCRIPTION
**Pull request by raaymax**

### CODE REVIEW

**Comment by FabienArcellier** [comment]
I didn't find a way to move to the chat page without using the builder. I would suggestion to add a link from the main page to the chat page.

File: apps/hello/main.py [hunk 1]
```diff
@@ -80,6 +80,75 @@ def _get_story_text():
...
+
+    elif payload == "highlight" and state["chat_bot"]["show_pdf"]:
+        state["chat_bot"]["pdf"]["highlights"] = ["FeatherByte", "SynthoCorp"];
+        return "I have highlighted some interesting parts of the story."
+    elif payload == "help":
+        return "You can type `pdf` or `web` to see what this components can do."
+    else:
+        return "I don't understand that command. Type 'help' to see what is possible"
+
+def _show(name, resource, state):
```
Comment (ramedina86) [inline apps/hello/main.py:113]: Can you change for a more descriptive method name, e.g. `_show_chatbot_resource`?

File: apps/hello/main.py [hunk 2]
```diff
@@ -80,6 +80,75 @@ def _get_story_text():
...
+                "desc": "Click to be open",
+                "data": "open_web" 
+            }]
+        }
+
+    elif payload == "highlight" and state["chat_bot"]["show_pdf"]:
+        state["chat_bot"]["pdf"]["highlights"] = ["FeatherByte", "SynthoCorp"];
+        return "I have highlighted some interesting parts of the story."
+    elif payload == "help":
+        return "You can type `pdf` or `web` to see what this components can do."
```
Comment (ramedina86) [inline apps/hello/main.py:109]: Should read "these"

File: apps/hello/main.py [hunk 3]
```diff
@@ -80,6 +80,75 @@ def _get_story_text():
...
+            }]
+        }
+    if payload == "open_web_components":
+        _show("web", "https://streamsync.cloud/component-list.html", state)
+        return "You can find all components here."
+    if payload == "open_web_tutorial":
+        _show("web", "https://www.streamsync.cloud/component-list.html", state)
+        return "You can find quick start tutorial here."
+
+    return "I don't understand that command. Type 'help' to see what is possible"
```
Comment (ramedina86) [inline apps/hello/main.py:150]: Missing period (.) at the end

File: apps/hello/main.py [hunk 4]
```diff
@@ -5,10 +5,10 @@
 import statistics
 
 from streamsync.core import StreamsyncState
+from streamsync.ui import StreamsyncUIManager
```
Comment (ramedina86) [inline apps/hello/main.py:8]: Are we using CMCs?

**Review by ramedina86 — COMMENTED** [review]
Awesome chat indeed; some comments about readability/consistency/grammar (the usual minor stuff)

File: apps/hello/main.py (hunk 4 above)
Comment (raaymax) [reply apps/hello/main.py:8]: I was using it at first but removed it in the end.

**Pull request merged**